src/components/connection-prompt/ConnectDialogContainer.svelte

```diff
@@ -166,13 +166,13 @@
 </script>
 
 <div bind:this={dialogContainer}>
-  <StandardDialog
+  <MeltDialog
     isOpen={$connectionDialogState.connectionState !== ConnectDialogStates.NONE &&
       !endOfFlow}
     onClose={connectionStateNone}
     hasCloseButton={$connectionDialogState.connectionState !==
       ConnectDialogStates.USB_DOWNLOADING}
-    dismissOnClickOutside={false}>
+    createDialogPropsOverride={{ closeOnOutsideClick: false, closeOnEscape: false }}>
```

Comment:
The dialog `heading` prop could not be used since the different contents of the connection dialog were being rendered by the same dialog component